src/app/components/summary/summary.component.css

```diff
@@ -0,0 +1,12 @@
+.right{
+  margin-left: 20px;
+}
+.left{
+  max-width: 20px;
+  min-width: 20px;
```

Comment:
wouldn't setting just `width` (and `overflow` if needed) have the same effect?

Reply:
yup, probably - I will try to improve this 👍